### DISCUSSION TITLE
windows-2022 contains a buggy VS version

### DISCUSSION THREAD

**Discussion by alexander-vitchevsky** [actions/virtual-environments#10073] [Installed software]
windows-2022 image version 20240603.1.1 contains an update of Visual Studio Enterprise 2022 into version 17.10.34928.147. This version has a bug described and discussed here:
https://developercommunity.visualstudio.com/t/Cannot-build-any-project-after-update-to/10664111

Bug was fixed by the next VS service release 17.10.2 (17.10.35004.147). 

Is it possible to expedite update of the pipeline to have a fixed VS version?

**Comment by subir0071** [comment]
We are in the process of rolling out visual studio version `17.14.5` this week, for windows-2022 runners.

**Comment by kishorekumar-anchala** [comment]
Hi @alexander-vitchevsky - kindly verify now , the new version is available with the latest image.
